test/integration/user_creation_test.rb

```diff
@@ -363,7 +307,7 @@ def test_user_create_openid_failure
   end
 
   def test_user_create_openid_redirect
-    auth_uid = "http://localhost:1123/new.tester"
+    auth_uid = "12345654321"
```

Comment:
I'm not sure why you're changing this to something that doesn't look like an OpenID id? You didn't change it in the `success` test?

Reply:
The reason is to have different values for `auth_uid` and `openid_url`, to be able to verify exactly which one goes where bellow in `assert_redirected_to` and `post "/user/new"`.

I think I should probably update `success` test as well.

Reply:
Updated `success` test case as well. Please let me know if it should be reverted to assume same values for `openid_url` and `auth_uid`

Reply:
I would have stuck with URLs and just made them different from each other but it probably doesn't really matter.